doxia-modules/doxia-module-markdown/src/main/java/org/apache/maven/doxia/module/markdown/LastTwoLinesBufferingWriter.java

```diff
@@ -91,4 +91,8 @@ public void flush() throws IOException {
     public void close() throws IOException {
         out.close();
     }
+
+    public boolean isAfterDigit() {
+        return currentLine.length() > 1 && Character.isDigit(currentLine.charAt(currentLine.length() - 1));
+    }
```

Comment:
This also covers non-European digits. Is that expected?

Reply:
The markdown spec is not very clear in this regard and also different markdown parses behave differently, therefore it is safer to also consider those as digits (i.e. escape rather too often than miss one time).